### PR TITLE
throw exception when stop loadbalance to return 503

### PR DESCRIPTION
**Pull request by dorisd0102**
Commit Message: throw exception when stop loadbalance to return 503
Additional Description:
Risk Level: Low
Testing: Done
Docs Changes: None
Release Notes: None
Platform Specific Features: None
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by dorisd0102** [comment]
@durd07  please help to view this change. Thanks.

**Comment by durd07** [comment]
LGTM

**Review by durd07 — DISMISSED** [review]

File: contrib/sip_proxy/filters/network/source/conn_manager.cc [hunk 1]
```diff
@@ -214,11 +214,17 @@ void ConnectionManager::continueHandling(MessageMetadataSharedPtr metadata,
   } catch (const AppException& ex) {
     ENVOY_LOG(debug, "sip application exception: {}", ex.what());
     sendLocalReply(*(decoder_->metadata()), ex, false);
+
+    std::string&& k = std::string(decoder_->metadata()->transactionId().value());
```
Comment (lizan) [inline contrib/sip_proxy/filters/network/source/conn_manager.cc:218]: This could be just absl::string_view, no need to be rvalue reference.

**Comment by dorisd0102** [comment]
@lizan please help to review rework version. Thanks.

the presubmit test failed should not be cause by this PR. 
2022-06-15T03:52:47.2050540Z [31m[1mFAILED: [0m//test/integration:protocol_integration_test (Summary)
2022-06-15T03:52:47.2053200Z       /private/var/tmp/_bazel_runner/3d1a7f52fd47fc9a8db71770aacd13e5/execroot/

**Review by lizan — APPROVED** [review]

**Pull request merged**